acslib/ccure/search.py

```diff
@@ -73,25 +76,29 @@ def filter(self, search):
 
 class PersonnelFilter(BaseCcureFilter):
     """Basic CCure Personnel Filter
-    :param lookups: List of tuples containing the field name and the lookup function
+    :param lookups: Dict containing searchable field names and their lookup functions
     :param outer_bool: Boolean operator to use between search terms
     :param inner_bool: Boolean operator to use between lookups
     :param term_operator: Term operator to use between field and a search term
+    :param display_properties: List of properties from CCure to be included in the CCure response
     :attribute
     """
 
     def __init__(
         self,
-        lookups: dict[str, callable] = [],
+        lookups: Optional[dict[str, callable]] = None,
         outer_bool=BooleanOperators.AND,
         inner_bool=BooleanOperators.OR,
         term_operator=TermOperators.FUZZY,
+        display_properties: Optional[list[str]] = None
     ):
-        self.filter_fields = lookups if lookups else PERSONNEL_LOOKUP_FIELDS
+        self.filter_fields = lookups or PERSONNEL_LOOKUP_FIELDS
         self.outer_bool = f" {outer_bool.value} "
         self.inner_bool = f" {inner_bool.value} "
         self.term_operator = term_operator.value
-        self.display_properties = ["FirstName", "MiddleName", "LastName"]
+        self.display_properties = display_properties
+        if display_properties is None:
```

Comment:
actually the reverse of this:

```python

self.display_properties = ["FirstName", "MiddleName", etc...]
if display_properties:
    self.display_properties = display_properties
```

Reply:
Just pushed an update. One small change to the example above: An empty list of display_properties means it'll print every property, which should be allowed, so I don't want to trigger the default unless `display_properties` is `None`

Reply:
I'm so sorry I missed this.

Reply:
All good! 👍